src/components/EventChat/CommentCard.tsx

```diff
@@ -184,10 +184,9 @@ const CommentCard = (props: EventCardProps) => {
             onChange={handleCommentMsgChange}
             multiline={true}
             rows="3"
+            placeholder="Add text..."
             className={classes.textField}
-            classes={{
-              root: classes.input,
-            }}
+            variant="filled"
```

Comment:
Make sure this is using the background color of `DARK`

Reply:
It is, but filled applies some grey over it so it isn't as dark.